# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1260556698 18000
#      Fri Dec 11 13:38:18 2009 -0500
# Node ID d4dab9605c49b53dee6e8453b71628433c83c80d
# Parent  a7e288044ef7ae347c20ab2cac245883426dd23a
correctly set indent-tabs-mode for ChangeLogs and Makefiles in .dir-locals.el

diff --git a/.dir-locals.el b/.dir-locals.el
--- a/.dir-locals.el
+++ b/.dir-locals.el
@@ -1,4 +1,5 @@
 ((nil . ((indent-tabs-mode . nil)
          (fill-column . 72)))
  (c-default-mode "gnu")
- ("ChangeLog" . ((nil . ((indent-tabs-mode . t))))))
+ (change-log-mode . ((indent-tabs-mode . t)))
+ (makefile-mode . ((indent-tabs-mode . t))))
diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,10 +1,13 @@
 2009-12-11  John W. Eaton  <jwe@octave.org>
 
+	* .dir-locals.el: Correctly set indent-tabs-mode for ChangeLogs,
+	and Makefiles.
+
 	* common.mk (cp_update_rule): New macro.
 
 2009-12-10  John W. Eaton  <jwe@octave.org>
 
 	* mkoctfile.in, mkoctfile.cc.in: If output file is specified and
 	it does not end in output_ext, append output_ext.
 
 2009-12-10  John W. Eaton  <jwe@octave.org>
